test_pycalculator.py

```diff
@@ -0,0 +1,48 @@
+import pytest
+import pycalculator
+import pickle
+import os
+
+
+@pytest.fixture()
+def temp_file(tmpdir):
+    """
+    :return: A path of a temp file that can be overwritten
+    """
+    return os.path.join(tmpdir, 'temp_file.temp')
+
+
+@pytest.mark.parametrize('dividend, divisor, quotient',
+                         ((10, 2, 5)
+                          , (-4, 2, -2)
+                          , (4, -2, -2)
+                          , (-5, -5, 1)
+                          , (0, 5, 0)
+                          , (2, 4, 0.5)))
+def test_div(dividend, divisor, quotient):
+    assert pycalculator.div(dividend, divisor) == quotient
+
+
+@pytest.mark.parametrize('num1, num2, the_sum',
+                         ((5, 10, 15)
+                          , (-2, 2, 0)
+                          , (5, -2, 3)
+                          , (-2, -3, -5)))
+@pytest.mark.xfail
```

Comment:
Why xfail?

Reply:
They said to mark the functions that don't work as xfail, but I wrote the functions so they work.
So I arbitrary chose 2 functions and marked them as xfail

Reply:
Another use for xfail other than marking bugs that will only be fixed in the future is to mark currently not supported features.
Add one of those.

Reply:
`xfail` should be removed once it doesn't fail anymore, so you can remove these.